core/src/main/java/hudson/model/MultiStageTimeSeries.java

```diff
@@ -169,7 +169,7 @@ public Api getApi() {
         public DateFormat createDateFormat() {
             switch (this) {
             case HOUR:  return new SimpleDateFormat("MMM/dd HH");
-            case MIN:   return new SimpleDateFormat("HH:mm");
+            case MIN:   return new SimpleDateFormat("E HH:mm");
```

Comment:
Is there a risk that `E` generates the same string for two weekdays in some locale?

I'm not very familiar with Java date formatting, but Unicode locale data supports "narrow" forms of weekdays that need not be unique: [Unicode Technical Standard #35 §2.1](https://www.unicode.org/reports/tr35/tr35-dates.html#months_days_quarters_eras), [Days - narrow - Formatting](https://unicode-org.github.io/cldr-staging/charts/latest/by_type/date_&_time.gregorian.html#Days_-_narrow_-_Formatting). The Windows NLS API likewise has [LOCALE_SSHORTESTDAYNAME*](https://docs.microsoft.com/windows/win32/intl/locale-sshortestdayname-constants) as described in [We call 'em ShortestDayNames for a reason; they're as short as they're EVER gonna be!](http://archives.miloush.net/michkap/archive/2011/12/30/10251957.html)

Reply:
Don't know, but perhaps not in practice? This always prints `7` for me:

```
Locale.getAvailableLocales().each { l ->
  def sdf = new java.text.SimpleDateFormat("E", l)
  def names = new HashSet<String>()
  for (int i = 0; i < 7; i++) {
    def dayName = sdf.format(new Date(122, 1, i+1))
//    println dayName
    names.add(dayName)
  }
  println names.size() + " in " + l
}
return
```